azure-pipelines.yml

```diff
@@ -0,0 +1,80 @@
+stages:
+ - stage: check
+   displayName: Compilation check
+   dependsOn: []
+   jobs:
+     - template: azure/cargo-check.yml@templates
+       parameters:
+         name: cargo_check
+ # This represents the minimum Rust version supported.
+ # Tests are not run as tests may require newer versions of rust.
+ - stage: msrv
+   displayName: "Minimum supported Rust version"
+   dependsOn: []
+   jobs:
+     - template: azure/cargo-check.yml@templates
+       parameters:
+        rust: 1.34.0
+ - stage: custom
```

Comment:
Should this stage maybe also include a check that the benchmarks compile?

Reply:
Ah, if you want that I'd just add `benches: true` under `parameters:` to the very first stage (`check`).

Reply:
Eventually it would be cool to save Criterion's benchmark reports and do some performance regression reporting for PRs, but that's definitely a stretch goal...